instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+// Components
+import SearchBar from '../SearchBar/SearchBar';
+import Posts from './Posts';
+
+// Data
+import dummyData from '../../dummy-data';
+
+class PostsPage extends Component {
+    constructor() {
+        super();
+
+        this.state = {
+            posts: [],
+            searchText: ''
+        }
+    }
+
+    componentDidMount() {
+        this.setState({
+            posts: dummyData
+        })
+    }
+
+    handleChange = e => {
```

Comment:
Yeah this is good.  Notice how this pattern lets you control a child component from here.  If you were to change `searchText` on your state manually it would update the value of the input field.  This is called 2-way-binding.